content/00.front-matter.md

```diff
@@ -4,9 +4,21 @@
 
 {% for author in authors %}
 + **{{author.full_name}}**<br>
-  [\@{{author.github_username}}](https://github.com/{{author.github_username}}) ·
-  [{{author.orcid}}](https://orcid.org/{{author.orcid}}) ·
-  {{author.affiliations}} ·
-  Funded by {{author.funders}}
+  {% set previous = false %}
```

Comment:
The `previous` flag makes this code block more complicated.  Could we simplify the code if we put some required information first, such as the affiliation?  That would also involve making an affiliation required in addition to the name and initials.

@dhimmel what do you think?